### PR TITLE
Listen to scroll on modal to update coordinates block coordinates

### PR DESCRIPTION
**Pull request by AlexandreCantin**

### CODE REVIEW

**Comment by AlexandreCantin** [comment]
review @vgrange :ferris_wheel:

File: frontend/src/components/companies/blocks/company_modal.js [hunk 1]
```diff
@@ -16,20 +20,25 @@ export class CompanyModal extends Component {
 
         this.hasSoftSkills = false;
         this.hasExtraInfos = false;
+        this.hasScrollEventListener = false;
+
+        this.updateCoordinatesFn = throttle(this.computeCoordinatesTopValue, 100);
```
Comment (vgrange) [inline frontend/src/components/companies/blocks/company_modal.js]: Throttle — invokes function at most once per every X milliseconds.

File: frontend/src/components/companies/blocks/company_modal.js [hunk 2]
```diff
@@ -78,13 +92,49 @@ export class CompanyModal extends Component {
...
+        // We got a scroll => Coordinates at bottom
+        if(modalHeight < modalScrollHeight + coordinatesHeight) {
+            let newTopValue = modalHeight + scrollTop - coordinatesHeight;
+            // If scrollbar is near to the bottom, we decrease the top value in order to see the company footer
+            if(modalHeight + scrollTop + FOOTER_HEIGHT_DELTA/2 >= modalScrollHeight) newTopValue -= FOOTER_HEIGHT_DELTA;
+
+            this.setState({ coordinateTop: { 'position': 'absolute', 'left': 0, 'top': newTopValue } });
+            this.forceUpdate();
+        } else {
+            // All the modal is visible ? We
```
Comment (vgrange) [inline frontend/src/components/companies/blocks/company_modal.js]: We what?

Comment (regisb) [reply frontend/src/components/companies/blocks/company_modal.js]: All we modal are belong to us

**Pull request merged**